umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md

```diff
@@ -0,0 +1,161 @@
+---
+description:
+  In this article we show how you can upgrade your Umbraco 10 Cloud project locally to Umbraco 11 and then migrate the project to a new Umbraco 11 project.
+---
+
+# Migrating a Umbraco 10 project to a Umbraco 11 project
+
+Due to a temporary bug in the hosting services, we do not recommend to directly upgrade to Umbraco 11.
```

Comment:
I don't think we need to mention this here. Wouldn't it make sense to instead keep the focus on the migration?

Reply:
Yeah makes sense, i am removing it!